src/DssSpell.sol

```diff
@@ -52,42 +66,50 @@ contract DssSpellAction is DssAction {
-                   ilk == "PSM-USDC-A"
-                || ilk == "ETH-B"
-                || ilk == "ETH-C"
-                || ilk == "RWA001-A"
-                || ilk == "RWA002-A"
-            )  {
-                continue;
-            }
-            DssExecLib.setIlkMinVaultAmount(ilk, 10_000);
-        }
-
-        // --------------------------- Add MCD_FLASH ---------------------------
-
-        Fileable(MCD_FLASH).file("max", 500_000_000 * WAD);
-        Fileable(MCD_FLASH).file("toll", 5 * WAD / 10000);
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-
-        // -------------------------- Update Chainlog --------------------------
 
-        DssExecLib.setChangelogVersion("1.9.1");
+        // ----------- Auto-Line updates -----------
+        // https://vote.makerdao.com/polling/QmZz4ssm?network=mainnet#poll-detail
+        DssExecLib.setIlkAutoLineParameters(ILK_PSM_USDC_A, 10_000_000_000, 1_000_000_000, 24 hours);
+
+        // ----------- Stability Fee updates -----------
+        // https://vote.makerdao.com/polling/QmfZWY87?network=mainnet#poll-detail
+        DssExecLib.setIlkStabilityFee("ETH-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("ETH-B", SIX_PCT, true);
+        DssExecLib.setIlkStabilityFee("ETH-C", ZERO_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("WBTC-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("LINK-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("YFI-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNI-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("AAVE-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("RENBTC-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("COMP-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("BAL-A", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", ONE_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIUSDC-A", ZERO_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2ETHUSDT-A", TWO_PCT, true);
+
+        // Core Unit Budget Distributions - July
+        DssExecLib.sendPaymentFromSurplusBuffer(GRO_MULTISIG,    126_117);
+        DssExecLib.sendPaymentFromSurplusBuffer(MKT_MULTISIG,     44_375);
+        DssExecLib.sendPaymentFromSurplusBuffer(GOV_MULTISIG,    273_334);
+        DssExecLib.sendPaymentFromSurplusBuffer(RWF_MULTISIG,    155_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(RISK_MULTISIG,   182_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(PE_MULTISIG,     510_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(ORA_MULTISIG,    419_677);
+        DssExecLib.sendPaymentFromSurplusBuffer(ORA_ER_MULTISIG, 800_000);
```

Comment:
What's the source of truth for these? Github? Forum posts?

Reply:
I'm working off a pre-print from GovAlpha that's being vetted by the respective CU facilitators. These values will need to be reconfirmed against the final copy. You'd want to check the respective MIP proposals for independent confirmation.

Reply:
I also managed to identity those numbers in the MIPS proposals both in the forum and in github (not sure which should be considered the souce).
It would be more effective to include a link in the comments to relevant documents.